src/polymer/js-utils.ts

```diff
@@ -46,27 +46,9 @@ export function toScannedPolymerProperty(
     }));
   }
 
-  let value;
-  if (babel.isClassMethod(node)) {
-    value = node;
-  } else {
-    value = node.value;
-  }
+  const value = babel.isObjectProperty(node) ? node.value : node;
 
-  let type;
-  if (value) {
-    type = closureType(value, sourceRange, document);
-  } else {
-    type = new Warning({
-      code: 'unknown-prop-type',
-      message:
-          `Could not determine type of property from expression of type: ` +
-          `${node.key.type}`,
-      sourceRange: sourceRange,
-      severity: Severity.INFO,
-      parsedDocument: document
-    });
-  }
+  let type = closureType(value, sourceRange, document);
```

Comment:
Why is this warning unnecessary?

Reply:
This diff might be misleading -- Master [doesn't have this](https://github.com/Polymer/polymer-analyzer/blob/master/src/polymer/js-utils.ts#L48), it was something I added because I had a scenario where node.value was undefined, but that scenario was because I didn't understand that the new ClassMethod and ObjectMethod both don't have value and that it should only be pulled from ObjectProperty.  After that, I was able to always have a defined node to send to closureType and no longer needed to pre-empt a call to it with my own Warning.  If you look at master today, the warning is not there.